docs/redoc.html

```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html>
```

Comment:
This seems like about all of the static code we should need to keep in VC from Redoc.